src/format.js

```diff
@@ -134,15 +134,34 @@ function addTag (meta, key, value, depth) {
 
       if (!Array.isArray(value) && depth < 2) {
         Object.keys(value).forEach(prop => {
-          addTag(meta, `${key}.${prop}`, value[prop], depth + 1)
+          addTag(span, meta, `${key}.${prop}`, value[prop], depth + 1)
         })
         break
       }
     default: // eslint-disable-line no-fallthrough
-      addTag(meta, key, serialize(value))
+      addTag(span, meta, key, serialize(value))
   }
 }
 
+function truncateText (span, key, value) {
+  if (typeof value !== 'string') {
+    log.error(
+      'Truncation at key=' +
+      key +
+      ' could not be performed for value type=' +
+      typeof value +
+      ' . Truncation skipped.'
+    )
```

Comment:
I don't think we should log anything here. May be a debug if we really want it but I don't see the need.

Reply:
removed, because.we anticipate other types now